docs/track.rst

```diff
@@ -1119,7 +1119,24 @@ With the operation ``create-snapshot`` you can `create a snapshot <https://www.e
 * ``request-params`` (optional): A structure containing HTTP request parameters.
 
 .. note::
-    When ``wait-for-completion`` is set to ``true`` Rally will report the achieved throughput in byte/s.
+    It's not recommend to rely on ``wait-for-completion=true``. Instead you should keep the default value (``False``) and use an additional ``wait-for-snapshot-create`` operation in the next step.
+    This is mandatory on the `Elastic Cloud <https://www.elastic.co/cloud>`_ or environments where Elasticsearch is sitting behind a network element that may terminate the blocking connection after a timeout.
```

Comment:
* on the Elastic Cloud -> on Elastic Cloud?
* "sitting behind a network element" -> "connected via intermediate network components, such as proxies, that may terminate ..."?

Reply:
Addressed in 32ee4abf6a15d4158992c9843958b51e9a766cb5